modules/packages/RangeChunk.chpl

```diff
@@ -0,0 +1,223 @@
+/*
```

Comment:
This new package module needs chpldoc comments and whatever build changes are necessary to generate docs for it.